P3826/P3826.md

```diff
@@ -1984,14 +1993,11 @@ subsection as follows:]{.ednote}
 >
 >     ::: add
 >
->     For a subexpression `env`, the lambda expression above returns an object `attrs` such
+>     For a subexpressions `tag` and `env`, the lambda expression above returns an object `attrs` such
 >     that
 >
->     - [10.1]{.pnum} `get_completion_domain<set_value_t>(attrs, env)` is expression-equivalent
->       to ```@_`when-all-domain`_@(env)```.
->
->     - [10.2]{.pnum} [TODO: more?]{.blue}
->
+>     - [10.1]{.pnum} `get_completion_domain<tag>(attrs, env)` is expression-equivalent
+>       to ```@_`when-all-domain`_@(tag, env)```.
```

Comment:
the tparam to `get_completion_domain` is a type, not a subexpression. also, should be bullet 10.2.